### ISSUE TITLE
Suivi état commande

### ISSUE THREAD

**Issue by pierrelouisgrangier** [pierrelouisgrangier/super#11]
En tant que client, je veux pouvoir suivre l'état de ma commande, afin de savoir quand elle sera livrée et planifier en conséquence.

une page en deux partie la droite et la gauche
la gauche doit avoir une liste d'état de de la commande de la forme suivante

- [ ] Payée

- [ ] En Préparation

- [ ] Expédié

- [ ] En cours de Livraison

- [ ] Livrée

chaque case doit avoir une couleur représentant l'état de l'étape en cours rouge orange et vert pour finie, en cours et a venir

la droite doit pouvoir contenir une liste d'information (pour le moment nous ne mettrons que la datte d'arrivé de la commande) 

on veut deux manière d'accès : un lien avec le numéros de commande et avec un bouton "Suivre mon colis" dans la page Mes commandes (qui existe déjà)

**Comment by LeGrandDevin** [comment]
@pierrelouisgrangier quelle element d'UI on s'attend?

**Comment by pierrelouisgrangier** [comment]
> @pierrelouisgrangier quelle element d'UI on s'attend?

On s'attend à avoir une liste sous la forme suivante :

- [ ] Payée
- [ ] En Préparation
- [ ] Expédié
- [ ] En cours de Livraison
- [ ] Livrée

**Comment by LeGrandDevin** [comment]
trés bien on veux pouvoir cliqué sur chaque élément?

**Comment by pierrelouisgrangier** [comment]
Non simplement avoir l'apparçu d'où est notre commande.

**Comment by LeGrandDevin** [comment]
ok

**Comment by LeGrandDevin** [comment]
donc tu le confirme
on veut une page permettant d'avoir le suivi de la commande avec 5 étape possible
Payée

En Préparation

Expédié

En cours de Livraison

Livrée

 lister les une en dessous des autres et précisé qi chaque étape et : finie, en cours et a venir

**Comment by pierrelouisgrangier** [comment]
C'est bien ça

**Comment by LeGrandDevin** [comment]
@pierrelouisgrangier uniquement les utilisateur connecter ou peut-on accéder a cette page a l'aide d'un code de commande?

**Comment by pierrelouisgrangier** [comment]
On peut y accéder grâce à un lien contenant le numéro de commande et un  bouton "Suivre mon colis" dans la page Mes commandes (qui existe déjà)

**Comment by LeGrandDevin** [comment]
les cases doivent pouvoir être cochés ou de différentes couleurs pour représenté l'état?

**Comment by LeGrandDevin** [comment]
@pierrelouisgrangier  Est-ce que l'on veut des informations complémentaires? Comme le nom de la société de transport l'adresse de livraison pou autre en plus de la date prévue d'arrivé?

**Comment by LeGrandDevin** [comment]
est ce que les information complémentaire sont présenter sous la forme d'une liste avec des information vide puis remplie au fur a mesure des étape du style 

date de livraison : 15/10/2024
date d'envoie 15/08/2024
compagnie de livraison: UPS
arivé de la commande :
état de la commande a l'arrivé :

**Comment by pierrelouisgrangier** [comment]
> cases doivent pouvoir être cochés ou de différentes couleurs pour représenter l'état?

Vert si passé et Rouge si pas encore passé.


> @pierrelouisgrangier Est-ce que l'on veut des informations complémentaires? Comme le nom de la société de transport l'adresse de livraison pou autre en plus de la date prévue d'arrivé?

Non, c'est la première phase on veut quelque chose de simple.

**Comment by LeGrandDevin** [comment]
OK on veut quand même la date de livraison ducoup? et pour le statut en cours on met jaune orange...

**Comment by pierrelouisgrangier** [comment]
> OK on veut quand même la date de livraison ducoup? et pour le statut en cours on met jaune orange...

On met en orange. Oui on peut mettre la date de livraison une fois la commande livrée.

**Comment by LeGrandDevin** [comment]
mais pour la user story on as : "En tant que client, je veux pouvoir suivre l'état de ma commande, afin de savoir quand elle sera livrée et planifier en conséquence."
ça ne répondras pas a la demande non?

**Comment by pierrelouisgrangier** [comment]
> mais pour la user story on as : "En tant que client, je veux pouvoir suivre l'état de ma commande, afin de savoir quand elle sera livrée et planifier en conséquence." ça ne répondras pas a la demande non?

Effectivement, j'avais oublié. Si on peut avoir la date estimée du transporteur alors on aimerait la mettre.

**Comment by LeGrandDevin** [comment]
a quelle emplacement au niveau de la page?

**Comment by LeGrandDevin** [comment]
on aurais selon ce que je vois la page diviser en deux partie avec d'un coté (a gauche) l'état de la commande et a droite les information complémentaire qui contiendrait pour le moment uniquement la date de livraison des que connue est ce bon?

**Comment by pierrelouisgrangier** [comment]
> on aurais selon ce que je vois la page diviser en deux partie avec d'un coté (a gauche) l'état de la commande et a droite les information complémentaire qui contiendrait pour le moment uniquement la date de livraison des que connue est ce bon?

Très bien
